Rakefile

```diff
@@ -1 +1,95 @@
-require "bundler/gem_tasks"
+require 'bundler/gem_tasks'
+require 'rake/clean'
+require 'cucumber/rake/task'
+require 'mechanize'
+require 'fileutils'
+require 'yaml'
+
+CDK_DOWNLOAD_URL='https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software'
```

Comment:
-1 I don't think the Rakefile size is an issue for now. The more files the more files the harder it becomes to piece everything together. 

A valid argument might be to extract this into some sort of properties file to be able to change these settings w/o affecting the _Rakefile_, but I don't think it is worth the effort here. I would suggest we wait and see how this shapes up.